deploy/podman/flightctl-pam-issuer/flightctl-pam-issuer-config/config.yaml.template

```diff
@@ -4,8 +4,8 @@ service:
   baseUIUrl: https://{{.global.baseDomain}}:443
 auth:
   pamOidcIssuer:
-    address: ":8444"
-    issuer: {{if .global.auth.pamOidcIssuer.issuer}}{{.global.auth.pamOidcIssuer.issuer}}{{else}}https://{{.global.baseDomain}}:8444/api/v1/auth{{end}}
+    address: {{if .global.auth.pamOidcIssuer.address}}{{.global.auth.pamOidcIssuer.address}}{{else}}:8444{{end}}
+    issuer: {{if .global.auth.pamOidcIssuer.issuer}}{{.global.auth.pamOidcIssuer.issuer}}{{else}}https://{{.global.baseDomain}}:{{if .global.auth.pamOidcIssuer.port}}{{.global.auth.pamOidcIssuer.port}}{{else}}8444{{end}}/api/v1/auth{{end}}
```

Comment:
Is port a new field ? If yes - it is not documented.
Why is the change to issuer is needed ? If someone wants to set a port he can set it in the issuer